Error while building

I am  trying to use this with honggfuzz's instructions - cding into the repo and running make, but I then encounter this error:

> /home/ben/repos/fuzzing/honggfuzz/qemu_mode/honggfuzz-qemu/linux-user/syscall.c: In function ‘do_syscall1’:
> /home/ben/repos/fuzzing/honggfuzz/qemu_mode/honggfuzz-qemu/linux-user/syscall.c:7661:30: error: implicit declaration of function ‘stime’; did you mean ‘utime’? [-Werror=implicit-function-declaration]
>  7661 |             return get_errno(stime(&host_time));
>       |                              ^~~~~
>       |                              utime
> /home/ben/repos/fuzzing/honggfuzz/qemu_mode/honggfuzz-qemu/linux-user/syscall.c:7661:30: error: nested extern declaration of ‘stime’ [-Werror=nested-externs]
> cc1: all warnings being treated as errors
> make[1]: *** [/home/ben/repos/fuzzing/honggfuzz/qemu_mode/honggfuzz-qemu/rules.mak:69: linux-user/syscall.o] Error 1
> make: *** [Makefile:472: i386-linux-user/all] Error 2

--- a/linux-user/syscall.c
+++ b/linux-user/syscall.c
@@ -7655,10 +7655,12 @@ static abi_long do_syscall1(void *cpu_env, int num, abi_long arg1,
 #ifdef TARGET_NR_stime /* not on alpha */
     case TARGET_NR_stime:
         {
-            time_t host_time;
-            if (get_user_sal(host_time, arg1))
+            struct timespec ts;
+            ts.tv_nsec = 0;
+            if (get_user_sal(ts.tv_sec, arg1)) {
                 return -TARGET_EFAULT;
-            return get_errno(stime(&host_time));
+            }
+            return get_errno(clock_settime(CLOCK_REALTIME, &ts));
         }
 #endif
 #ifdef TARGET_NR_alarm /* not on alpha */

